cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -257,6 +257,8 @@ void CobTwistController::reconfigureCallback(cob_twist_controller::TwistControll
     twist_controller_params_.limiter_params.enforce_vel_limits = config.enforce_vel_limits;
     twist_controller_params_.limiter_params.enforce_acc_limits = config.enforce_acc_limits;
     twist_controller_params_.limiter_params.limits_tolerance = config.limits_tolerance;
+    twist_controller_params_.limiter_params.max_lin_vel = config.max_vel_lin_base;
+    twist_controller_params_.limiter_params.max_lin_rot = config.max_vel_rot_base;
```

Comment:
do not use `config.max_vel_lin_base`/`config.max_vel_rot_base` as these are the limits for the BaseActive extension, i.e. output limits for Cartesian twist to the base!
you need to add new parameters to dynamic reconfigure for limiting the input Cartesian velocity for the endeffector!